Source/Core/DolphinWX/Frame.cpp

```diff
@@ -1458,7 +1458,7 @@ void CFrame::ParseHotkeys()
 		VertexShaderManager::ResetView();
 
 	// Savestates
-	for (int i = 0; i < State::NUM_STATES; i++)
+	for (unsigned i = 0; i < State::NUM_STATES; i++)
```

Comment:
Not important, but related: looks like there's not much of a good reason for IsHotkey to take a signed 'id' parameter afaict?

It passes it to HotkeyManagerEmu::IsPressed which pretty much treats it as unsigned.